### PR TITLE
Finish_chapter03

### PR DESCRIPTION
**Pull request by tranminhvuong**
screenshot from 2018-11-23 16-18-11

### CODE REVIEW

**Comment by tranminhvuong** [comment]
ready

**Comment by tranminhvuong** [comment]
Ready

File: .bundle/config [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (DuongNT1512) [inline .bundle/config]: em tách riêng 3 pull
pull 1: init project
   Lưu ý trong pull này ignore forlder .bundle vs .ideal nhé
pull 2: init rubocop
pull 3: chapter3 - Mostly static pages

**Comment by DuongNT1512** [comment]
commented

**Pull request closed without merging**